src/sonic-frr/patch/0030-bgpd-error-handling-improve.patch

```diff
@@ -0,0 +1,42 @@
+From 33a231ac26a46bfb728975d1695b7a71a7bc95ab Mon Sep 17 00:00:00 2001
+From: stormliang <stormliang@microsoft.com>
```

Comment:
is this upstream patch?

Reply:
No, I this is not upstream patch, I create this one.